pkg/util/log/strip.go

```diff
@@ -83,7 +83,7 @@ func matchYAMLKey(key string) *regexp.Regexp {
 // matchYAMLKeyEnding returns a regexp matching a single YAML line with a key ending by the string passed as argument.
 // The returned regexp catches only the key and not the value.
 func matchYAMLKeyEnding(ending string) *regexp.Regexp {
-	return regexp.MustCompile(fmt.Sprintf(`(\s*(\w|_)*%s\s*:).+`, ending))
+	return regexp.MustCompile(fmt.Sprintf(`(^\s*(\w|_)*%s\s*:).+`, ending))
```

Comment:
Is that an intended change?

Reply:
Yes. Problem was that I can have an error message like `failure to get oath token: err here`, which would end up redacted.

Since the regex here is intended to match on yaml keys, I figured matching a beginning of line would not cause problems